src/Temporal/TemporalWorkerFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+namespace Baldinof\RoadRunnerBundle\Temporal;
+
+use Temporal\Worker\WorkerFactoryInterface;
+use Temporal\Worker\WorkerInterface;
+use Temporal\Worker\WorkerOptions;
+
+final class TemporalWorkerFactory
```

Comment:
Why is this class needed? It looks like it's doing the same thing as `WorkerFactoryInterface`?